sklearn/linear_model/coordinate_descent.py

```diff
@@ -766,9 +766,8 @@ def _decision_function(self, X):
         """
         check_is_fitted(self, 'n_iter_')
         if sparse.isspmatrix(X):
-            return np.ravel(safe_sparse_dot(self.coef_, X.T,
-                                            dense_output=True) +
-                            self.intercept_)
+            return safe_sparse_dot(X, self.coef_.T, dense_output=True) \
```

Comment:
i think its better / more in line with previous code to move the `dense_output=True` parameter to the next line and eliminate the need for the `\`.